openshift/Dockerfile.e2e-ansible

```diff
@@ -0,0 +1,20 @@
+FROM basebuilder AS builder
```

Comment:
It seems easier to maintain it like this. We won't have to update it on every branch cut and it will minimize the effort going forward. Also we just need the `basebulider` to re-generate the `testdata` directory. We are not building any binary which will be impacted by the base image.